zaza/openstack/utilities/generic.py

```diff
@@ -706,3 +706,23 @@ def attach_file_resource(application_name, resource_name,
         fp.flush()
         model.attach_resource(
             application_name, resource_name, fp.name)
+
+
+def get_leaders_and_non_leaders(application_name):
+    """Get leader node and non-leader nodes of percona.
+
+    Update and set on the object the leader node and list of non-leader
+    nodes.
+
+    :returns: None
```

Comment:
These two lines about the return value also are wrong. I know it's not your fault, you just moved this code, but let's use this opportunity to get this straight, thanks!

Reply:
done